### PR TITLE
Default npm algorithm = "flat"

### LINKED ISSUE #700: Add npmAlgorithm: nested
Since flat will become the default in 1.0.

### PR DESCRIPTION
**Pull request by pYr0x**
for steal 1.0 npm algorithm will be flat which means that npm 3 or higher is recommended.
close #700 #701

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "devDependencies": {
     "steal-css": "0.4.1",
     "steal-less": "0.2.2",
-    "system-npm": "0.6.16",
+    "system-npm": "stealjs/system-npm#major",
```
Comment (matthewp) [inline package.json]: I'll do a prerelease of this so we can use npm instead of git.

Comment (pYr0x) [inline package.json]: ok... i will wait for it ;)

**Comment by matthewp** [comment]
Just released system-npm `1.0.0-pre.0`, update the npm dependency to that.

**Comment by pYr0x** [comment]
looks good ... ready to merge.. feel free to merge it

**Pull request merged**